Form/Type/ResetPasswordType.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Webstack\UserBundle\Form\Type;
+
+use Rollerworks\Component\PasswordStrength\Validator\Constraints\PasswordStrength;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Security\Core\Security;
+use Symfony\Component\Validator\Constraints\NotCompromisedPassword;
+
+/**
+ * Class ResetPasswordType
+ */
+class ResetPasswordType extends AbstractType
+{
+    /**
+     * @var Security
+     */
+    private $security;
+
+    /**
+     * @var string
+     */
+    private $userClass;
+
+    /**
+     * ResetPasswordType constructor.
+     * @param Security $security
+     * @param string $userClass
+     */
+    public function __construct(Security $security, string $userClass)
+    {
+        $this->security = $security;
+        $this->userClass = $userClass;
+    }
+
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('password', RepeatedType::class, [
+            'label' => 'Wachtwoord',
+            'type' => PasswordType::class,
+            'options' => [
+                'attr' => [
+                    'autocomplete' => 'new-password',
+                ],
+            ],
+            'constraints' => [
+                new NotCompromisedPassword([
+                    'message' => 'Het ingevulde wachtwoord kan niet worden gebruikt omdat deze voorkomt op een lijst met gelekte wachtwoorden.',
+                ]),
+                new PasswordStrength([
+                    'minStrength' => 4,
```

Comment:
Zouden we deze instellingen (eventueel in een volgende PR) instelbaar kunnen maken

Reply:
Heb hem genoteerd, Ik zal dit opvolgen in een andere PR